src/main/java/core/stack/MateStack.java

```diff
@@ -1,19 +1,64 @@
 package core.stack;
 
+import java.util.EmptyStackException;
+
 public class MateStack<T> {
+    private int size;
+    private Node<T> head;
+
+    private static class Node<T> {
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents
better move it after public methods

Reply:
agree